crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -1,4 +1,7 @@
-use std::io::Write;
+use std::{
+    io::{self, Read, Write},
+    path::PathBuf,
+};
```

Comment:
style nit: I prefer
- Not using multi-line `use`s
- Minimizing the user of `use` generally (traits with `as _` being an exception)
  - Of these, the one I probably care most about is `use std::self` as that doesn't really buy us all that much

Reply:
Its been a while since I last looked at it and everything was unstable then...

Reply:
Looks like `imports_granularity` is not stable: https://rust-lang.github.io/rustfmt/?version=v1.5.1&search=#imports_granularity